cypress/integration/inventory/create-holdings-as-different-user.spec.js

```diff
@@ -0,0 +1,61 @@
+import TopMenu from '../../support/fragments/topMenu';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+import searchInventory from '../../support/fragments/data_import/searchInventory';
+import { MultiColumnListCell } from '../../../interactors';
+import permissions from '../../support/dictionary/permissions';
+import InventoryInstance from '../../support/fragments/inventory/inventoryInstance';
+import HoldingsRecordView from '../../support/fragments/inventory/holdingsRecordView';
+import getRandomPostfix from '../../support/utils/stringTools';
+import testTypes from '../../support/dictionary/testTypes';
```

Comment:
Minor comment: let's use same approach, capital letters for all imports. e.g. TestTypes instead of testTypes etc.